variables.tf

```diff
@@ -76,6 +76,18 @@ variable "name_static_nat_ips" {
   default     = "nat-manual-ip"
 }
 
+variable "nat_min_ports_per_vm" {
+  # https://cloud.google.com/nat/docs/ports-and-addresses#ports-and-connections
+  # The number of NAT source IP address and source port tuples that a Cloud NAT gateway reserves for a VM limits the number of connections that the VM can make to a unique destination:
+
+  # https://cloud.google.com/nat/docs/ports-and-addresses#port-reservation-examples
+  # Each nat ip supports 65536 ports
+  # If min ip per vm is 2048, then total number of vms that can be attached to the nat is (nat static ip count) * 65566 / 2048 = 32
+  description = "Minimum number of ports allocated to a VM from the NAT"
+  type        = number
+  default     = 64
+}
```

Comment:
Kindly put these comments inside the description itself so that it's rendered as part of the documentation in the TF registry.